File: app/src/main/java/com/moudle/app/BitmapDrawableActivity.java
package com.moudle.app;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.widget.LinearLayout;

/**
 * BitmapDrawable       bitmap 设置资源文件
 */
public class BitmapDrawableActivity extends AppCompatActivity {

    private static final String TAG = "BitmapDrawableActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_bitmap);

        LinearLayout lin = findViewById(R.id.lin);

        Log.e(TAG, "BitmapDrawableActivity.onCreate." + lin.toString());

    }
}

File: app/src/main/java/com/moudle/app/activity/BitmapDrawableActivity.kt
package com.moudle.app.activity

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import com.moudle.app.databinding.ActivityBitmapBinding

/**
 * BitmapDrawable       bitmap 设置资源文件
 */
class BitmapDrawableActivity : AppCompatActivity() {
    private lateinit var bitmapBinding: ActivityBitmapBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        bitmapBinding = ActivityBitmapBinding.inflate(layoutInflater)
        setContentView(bitmapBinding.root)
        Log.e(TAG, "BitmapDrawableActivity.onCreate.${bitmapBinding.lin}")
    }

    companion object {
        private const val TAG = "BitmapDrawableActivity"
    }
}